.gitignore

```diff
@@ -21,4 +21,4 @@ debug
 /mac_install.log
 /win_install.log
 /linux_install.log
-/src/default_excludes.mk
\ No newline at end of file
+/src/default_excludes.mk
```

Comment:
I'm not sure, I don't see gitignore listed in the changed files anymore. I think my latest commit (d74823075213b33198c723f2cc3c86c70f6c1cf1) returned the file back to normal